warehouse/metrics-core/src/main/java/datawave/metrics/analytic/FileByteSummaryLoader.java

```diff
@@ -105,11 +105,11 @@ public int run(String[] args) throws Exception {
         job.setMapOutputKeyClass(Key.class);
         job.setMapOutputValueClass(Value.class);
         job.setInputFormatClass(AccumuloInputFormat.class);
-        AccumuloInputFormat.setConnectorInfo(job, userName, new PasswordToken(password));
-        AccumuloInputFormat.setInputTableName(job, inputTable);
-        AccumuloInputFormat.setScanAuthorizations(job, Authorizations.EMPTY);
-        AccumuloInputFormat.setZooKeeperInstance(job, ClientConfiguration.loadDefault().withInstance(instance.trim()).withZkHosts(zookeepers.trim()));
-        AccumuloInputFormat.setRanges(job, Collections.singletonList(dayRange));
+
+        Properties clientProperties = Accumulo.newClientProperties().to(instance.trim(), zookeepers.trim()).as(userName, password).build();
```

Comment:
Can we disable formatting and put builder options on separate lines, like so?
```java
// @formatter:off
AccumuloInputFormat.configure()
    .clientProperties(clientProperties)
    .table(inputTable)
    .auths(Authorizations.EMPTY)
    .ranges(Collections.singletonList(dayRange))
    .store(job);
// @formatter:on
```

Reply:
Comment applies throughout.

Reply:
No problem, done.

Reply:
Thank you so much!

Reply:
No problem!